IPFS-Practical.md

```diff
@@ -92,6 +92,12 @@ In the same terminal now install `@openzeppelin/contracts` as we would be import
 npm install @openzeppelin/contracts
 ```
 
+If you are a Windows User, you'll have to add one more dependency. It is given below:
+
+```bash
+npm install --save-dev @nomicfoundation/hardhat-toolbox
```

Comment:
Let's move this line above the OpenZeppelin installation. Basically below the `...then follow the instructions on your terminal.` to that we keep things a bit organized.